### PR TITLE
SVG files, not strings.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes

Now that we're importing stylesheet assets using Webpack's URL loader, we no longer need to do extra work around manually creating data URIs for SVG icons... Webpack will automatically either resolve the path and add a hash to the filename, or inline it if it's under 8kb (which all these files are!)

---

For review: @DoSomething/front-end

### CODE REVIEW

File: scss/_regions/_modal.scss [hunk 1]
```diff
@@ -25,13 +25,15 @@
 // See http://www.github.com/DoSomething/modal
 [data-modal] {
   .heading.-banner {
-    // make space for "x" button and add proper left spacing
-    padding-left: $base-spacing;
-    padding-right: 72px;
+    // make space for "x" button on the right.
+    padding: $base-spacing 72px $base-spacing $base-spacing;
```
Comment (weerd) [inline scss/_regions/_modal.scss:29]: Is it supposed to have top and bottom padding? Not actually sure, based on new design changes.

Comment (DFurnes) [inline scss/_regions/_modal.scss:29]: Oh hah, didn't realize I included that commit in here. :blush: Yes, that's an adjustment based on the changes to the "banner" modifier in #536.

Comment (weerd) [inline scss/_regions/_modal.scss:29]: Cool cool. Just making sure!

**Comment by weerd** [comment]
:+1:

**Pull request merged**